src/validators/v1/organization.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	update: (req) => {
+		req.checkBody('mentee_feedback_question_set')
+			.notEmpty()
+			.withMessage('mentee_feedback_question_set is required')
+			.matches(/^[a-zA-Z0-9_]+$/)
```

Comment:
@sumanvpacewisdom it should be same validation, what we allow while creating question sets

Reply:
@rakeshSgr Its same , i have verified and added this